backend/tests/services/memory/test_manager.py

```
@@ -0,0 +1,186 @@
+    """Mock settings for testing."""
+    with patch("app.services.memory.manager.settings") as mock_settings:
+        mock_settings.MEMORY_ENABLED = True
+        mock_settings.MEMORY_BASE_URL = "http://localhost:8080"
+        mock_settings.MEMORY_API_KEY = "test-key"
+        mock_settings.MEMORY_TIMEOUT_SECONDS = 2.0
+        mock_settings.MEMORY_MAX_RESULTS = 5
+        yield mock_settings
+
+
+@pytest.fixture
+def memory_manager(mock_settings):
```

Ruff (0.14.11)

Unused function argument: `mock_settings`

(ARG001)